src/game/cheats.c

```diff
@@ -0,0 +1,39 @@
+#include "cheats.h"
+
+#include "building/type.h"
+#include "city/finance.h"
+#include "city/victory.h"
+#include "graphics/window.h"
+#include "scenario/invasion.h"
+#include "window/building_info.h"
+
+static struct {
+    int is_cheating;
+} data;
+
+void game_cheat_activate(void)
+{
+    if (window_is(WINDOW_BUILDING_INFO)) {
+        data.is_cheating = window_building_info_get_building_type() == BUILDING_WELL;
+    } else if (data.is_cheating && window_is(WINDOW_MESSAGE_DIALOG)) {
+        data.is_cheating = 2;
+        scenario_invasion_start_from_cheat();
+    } else {
+        data.is_cheating = 0;
+    }
+}
+
+void game_cheat_money(void)
+{
+    if (data.is_cheating) {
+        city_victory_force_win();
```

Comment:
You mixed up the money and victory cheat! Almost as good as Caesar firing you in the map editor haha :wink:

Reply:
I had too much sun yesterday 😂